bin/index.js

```diff
@@ -3,16 +3,13 @@
 require('dotenv').config();
 const http = require('http');
 const handler = require('../src/handler');
-const getConfig = require('../src/config');
+const parseEnv = require('../src/parse-env');
 
-let config;
-
-try {
-  config = getConfig(process.env);
-} catch (err) {
+process.on('uncaughtException', err => {
```

Comment:
I moved the CLI error handling to `uncaughtException` since it seemed simpler than catching both `parseEnv` and `http.createServer(handler(config))`. It'll also keep errors in a standard format even if they're thrown from the server itself.